sound/soc/intel/common/soc-acpi-intel-adl-match.c

```diff
@@ -260,6 +260,25 @@ static const struct snd_soc_acpi_link_adr adl_sdw_rt711_link2_rt1316_link01_rt71
 	{}
 };
 
+static const struct snd_soc_acpi_link_adr adl_sdw_rt711_link2_rt1316_link01[] = {
```

Comment:
commit message:

support configuration with RT1316 amplifiers on link0 and link1, and RT711 on link2.

This configuration does not rely on a dedicated RT715 chip for microphones, instead local microphones are connected to the RT711 chip, which handles both headset mic and local analog mics.

Reply:
@plbossart Sorry. I didn't describe it clearly. In this patch, the analog mics means headset mic, not the onboard analog mic.

Reply:
support configuration with RT1316 amplifiers on link0 and link1, and RT711 on link2 for headphone/headset. This product does not support local microphones.